mex-wholebodymodel/library/src/modelrototranslationmatrix.cpp

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright (C) 2016 Robotics, Brain and Cognitive Sciences - Istituto Italiano di Tecnologia
+ * Authors: Martin Neururer
+ * email: martin.neururer@gmail.com, gabriele.nava@iit.it
+ *
+ * The development of this software was supported by the FP7 EU projects
+ * CoDyCo (No. 600716 ICT 2011.2.1 Cognitive Systems and Robotics (b))
+ * http://www.codyco.eu
+ *
+ * Permission is granted to copy, distribute, and/or modify this program
+ * under the terms of the GNU General Public License, version 2 or any
+ * later version published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+ * Public License for more details
+ */
+
+// global includes
+
+// library includes
+#include <yarpWholeBodyInterface/yarpWholeBodyModel.h>
+
+// local includes
+#include "modelrototranslationmatrix.h"
```

Comment:
rototranslation matrix is not an english word, unfortunatly. : ( 
Do you care of changing any occurence of it to Homegeneous Transform Matrix or something similar?